lib/git.js

```diff
@@ -3,10 +3,19 @@
 let cp = require('child_process')
 
 module.exports = function (context) {
+  function wrapReject (reject) {
+    return function (error) {
+      if (error.code === 'ENOENT') {
```

Comment:
@dickeyxxx fixed in fb063ec and what do you mean by "heroku-git commands" as this is the `heroku-git` plugin, what is the plugin you want me to check?